settings/example-settings.php

```diff
@@ -206,18 +201,112 @@ function wpsf_tabless_settings( $wpsf_settings ) {
 					'external' => true, // Default is `true`.
 				),
 			),
+			array(
+				'id'      => 'more-text',
+				'title'   => 'More Text',
+				'desc'    => 'This is a description.',
+				'type'    => 'text',
+				'default' => 'This is default',
+			),
+			array(
+				'id'              => 'control-group',
+				'title'           => 'Control Group',
+				'subtitle'        => 'Selection option 1 or 2 to show and hide controls.',
+				'type'            => 'select',
+				'choices'         => array(
+					'option-1' => 'Option 1',
+					'option-2' => 'Option 2',
+					'option-3' => 'Option 3',
+				),
+				'default'         => 'text',
+				'show_controller' => 'control-group', // Needs to be set to the same as the control group. Does not have to match the Id of this control.
+			),
+			array(
+				'id'                 => 'show-if-option-1',
+				'title'              => 'Show if Option 1',
+				'subtitle'           => 'Will show if Option 1 is set.',
+				'type'               => 'select',
+				'type'               => 'text',
+				'default'            => 'This is default',
+				'show_control_group' => 'control-group', // Needs to be set to the control group.
+				'show_if_value'      => array( 'option-1' ), // show if will hide the control unless the value matches.
+
+			),
+			array(
+				'id'                 => 'show-if-option-2',
+				'title'              => 'Show if Option 2',
+				'subtitle'           => 'Will show if Option 2 is set.',
+				'type'               => 'select',
+				'type'               => 'text',
+				'default'            => 'This is default',
+				'show_control_group' => 'control-group', // Needs to be set to the control group.
+				'show_if_value'      => array( 'option-2' ), // show if will hide the control unless the value matches.
+			),
+			array(
+				'id'                 => 'show-if-option-2-or-3',
+				'title'              => 'Show if Option 2 or 3',
+				'subtitle'           => 'Will show if Option 2 or 3 is set.',
+				'type'               => 'select',
+				'type'               => 'text',
+				'default'            => 'This is default',
+				'show_control_group' => 'control-group', // Needs to be set to the control group.
+				'show_if_value'      => array( 'option-2', 'option-3' ), // show if will hide the control unless the value matches.
+			),
+			array(
+				'id'                 => 'hide-if-option-1',
+				'title'              => 'Hide if Option 1',
+				'subtitle'           => 'Will hide if Option 1 is set.',
+				'type'               => 'select',
+				'type'               => 'text',
+				'default'            => 'This is default',
+				'show_control_group' => 'control-group', // Needs to be set to the control group.
```

Comment:
I'm wondering if we should call these `field_control_group` just to match the format of tab and section?